test/test-pages/virginia-edu-broken-attributes/expected.html

```diff
@@ -0,0 +1,12 @@
+<div id="readability-page-1" class="page">
```

Comment:
This isn't actually very good, but it addresses the invalid markup breaking us entirely.